public/src/main/java/com/revenuecat/purchases/models/StoreProduct.kt

```diff
@@ -54,6 +54,7 @@ interface StoreProduct : Parcelable {
      */
     val defaultOption: SubscriptionOption?
 
+    // TODO javadocs
```

Comment:
Hmm I'm not sure how we ended up on this but I believe we should try to hide this field from the API if possible... So I would change the TODO to `remove from public API or something like that. Do lmk if I missed any conversations though!

Reply:
we added a TODO to reconsider visibility... i put this here in case we end up not hiding it. but i suppose it's redundant, so i can remove!